### PR TITLE
Adding BigQuery snippet for QueryParameters

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**

### CODE REVIEW

**Comment by garrettjonesgoogle** [comment]
Added @tswast in case he can get to this earlier than @lesv

File: google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/BigQuerySnippets.java [hunk 1]
```diff
@@ -610,6 +611,36 @@ public QueryResponse runQuery(String query) throws InterruptedException {
   }
 
   /**
+   * Example of running a query with query parameters.
+   */
+  // [TARGET query(QueryRequest)]
+  // [VARIABLE "SELECT distinct(corpus) FROM `bigquery-public-data.samples.shakespeare` where word_count > ?"]
```
Comment (tswast) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/BigQuerySnippets.java]: The BigQuery team would prefer we show named parameters if we had to pick just one.

File: google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/BigQuerySnippets.java [hunk 2]
```diff
@@ -610,6 +611,36 @@ public QueryResponse runQuery(String query) throws InterruptedException {
...
+  public QueryResponse runQueryWithParameters(String query) throws InterruptedException {
+    // [START runQueryWithParameters]
+    QueryRequest request = QueryRequest.newBuilder(query)
+        .setUseLegacySql(false) // standard SQL is required to use query parameters
+        .addPositionalParameter(QueryParameterValue.int64(5))
```
Comment (tswast) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/BigQuerySnippets.java]: Ditto. Let's use a named parameter.

**Comment by tswast** [comment]
Thanks

**Pull request merged**